src/toolchains.rs

```diff
@@ -238,8 +238,19 @@ impl Toolchain {
                     .join(&format!("target-{}", self.rustup_name())),
             );
         }
+        let mut script = cfg.args.script.clone();
+        if let Some(path) = &script {
+            if !path.is_absolute() && !path.starts_with("./") && !path.starts_with(".\\") {
+                if let Ok(mut dir) = std::env::current_dir() {
+                    dir.push(path);
+                    if dir.is_file() {
+                        script = Some(dir)
+                    }
+                }
+            }
+        }
```

Comment:
I think this could be simplified to something like:

```rust
let script = cfg.args.script.as_ref().map(|script| {
    std::env::current_dir().unwrap().join(script)
});
```

If the path is absolute, it will replace the current dir. Otherwise, it will be joined with it.